akka-actor-tests/build.sbt

```diff
@@ -7,3 +7,5 @@ Formatting.formatSettings
 publishArtifact in Compile := false
 
 Dependencies.actorTests
+
+AkkaBuild.dontPublishSettings
```

Comment:
Cool. You can check what autoplugins are enabled on what projects with `plugins` sbt command.